### DISCUSSION TITLE
Replacement for Mac OSX?

### DISCUSSION THREAD

**Discussion by terry9873** [helloSystem/hello#574] [General]
Hi. sorry to raise a topic here but not sure where else to turn/ask. 
Without boring you, I'm a Mac user for 15-20 years. I now DETEST Apple, mainly for their lies about privacy focus. Trouble is, I have tried Fedora (for a YEAR now) and just can't get used to Gnome and all its bugs (no preview file open dialogues, no columns Files browser, i miss that from Mac!). 
My attempt to leave Mac has cost me so much in terms of productivity, I know it so well. I am realising that Fedora just is slowing me down too much. It's so bad I am tempted (forced) to swallow my huge hatred of Apple and just go back to it. But someone sent me a link to this github page. I am not well-versed  on github, barely know how to navigate it, so can i just ask a few questions please:

Does this OS have column browser for Files app?
Is there a video somewhere where I can see this OS in use, to get a feel for what it offers, how it works etc?
thanks, and THANKS for doing this, whether I can use it or not. Apple is a disgrace.

**Comment by probonopd** [comment]
Hello @terry9873. I can really feel with you. 

> Does this OS have column browser for Files app?

No, not yet, but I have written a simple one for Windows, maybe one day I will find the time to port it to helloSystem (shouldn't be all too hard): https://github.com/probonopd/Miller?tab=readme-ov-file

> Is there a video somewhere where I can see this OS in use, to get a feel for what it offers, how it works etc?

Have you searched for "helloSystem" on YouTube?

> thanks, and THANKS for doing this, whether I can use it or not. Apple is a disgrace.

Thank you for your kind feedback. I don't think helloSystem can (or should) be a "replacement", but rather an _alternative_ that is hopefully more appealing to switchers coming from the Mac (of old) than, say, Gnome.

**Reply by terry9873** [reply]
Thanks very much. Yep I did look on Youtube but only found 1 or 2, and they seemed very old and just showed installing rather than using the OS. I just remembered I have a 2009 iMac in back of my office, was going to be a rifle target but I think I'll get one more use out of it first, playing with Hello :D 
Thanks, great work, and nice to see I am not as alone in disliking the privacy horror show that is now Apple, as anyone on the Apple forum would have me believe :)

**Reply by NitroNils** [reply]
The best way to get a feel of HelloSystem is to try it. Run it from a USB stick onto a machine. It is not finished, but gives a taste of where it's heading.
Also never tried it on Apple hardware yet.

Hope this is helpful and correct.

**Comment by athpanos** [comment]
You can also check RavynOS. It is in very early stages of development but it aims to be compatible with MacOS.
https://ravynos.com/
https://github.com/ravynsoft/ravynos

**Reply by terry9873** [reply]
Thanks, but I don't really understand what you mean by 'compatible' with MacOS. I am looking for a replacement for MacOS

**Reply by athpanos** [reply]
If you check the website you will understand. Their goal is to be able to run Mac applications.

From the github page:

A BSD-based OS project that aims to provide source and binary compatibility with macOS® and a similar user experience.

Source compatibility with macOS applications (i.e. you could compile a Mac application on ravynOS and run it)

Eventual compatibility with x86-64/arm64 macOS binaries (Mach-O) and libraries

**Reply by athpanos** [reply]
I believe it is what you are looking for. An OS that looks like MacOS and that it will be able to run Mac applications.

**Reply by terry9873** [reply]
Ah, if that's the case (running mac apps) then yes, worth keeping an eye on. 
I do wish people understood that only about 0.01% of the population understand phrases like "BSD-based OS project that aims to provide source and binary compatibility", it may as well be written in Hungarian for all I understand from that! (I do appreciate it's not written for non devs)

**Reply by eimfach** [reply]
But RavynOS looks really immature, how can it be an alternative already if it's far from being what OP needs: a productive and ready system ? I mean I appreciate their efforts but honestly, it looks far from being ready for productive use ?

**Reply by athpanos** [reply]
That is why I wrote that it is in early stages of development. To keep an eye on it in the future. RavynOS and hellosystem are the only two options to replace MacOS. Sadly, none is ready at the moment.

**Reply by terry9873** [reply]
As Eimfach said, I am looking for something current, but you said it's in development and that much is very clear reading about it, but I definitley bookmarked it to keep an eye on. 
Thanks all

**Comment by eimfach** [comment]
Maybe you could also try Haiku OS, I think it pretty much ready to use and has a very interesting approach.

**Reply by NitroNils** [reply]
Especially since they recently actually got Firefox running natively. (For those who didn't catch the news.)
